hedera-node/hedera-mono-service/src/main/java/com/hedera/services/store/models/Account.java

```diff
@@ -64,7 +64,7 @@
  * the Hedera Token Service. The memo field, for example, is not yet present.
  */
 public class Account {
-    private final Id id;
+    private Id id;
```

Comment:
Is there a reason to delete the `final` modifier?

Reply:
Yes, it's because in the lazy create scenario we need to [construct an `Account` instance with no `id` and only `alias` set](https://github.com/hashgraph/hedera-services/pull/4070/files#diff-c6c6cc6be40f874314cb722ed43c249ea1a005539bc103c49697d0baa2197cd7R123).